examples/memcached/cookbooks/custom-memcached/README.md

```diff
@@ -0,0 +1,121 @@
+# Memcached
+
+The memcached cookbook installs memcached and monitors it with monit. A `/data/app_name/shared/config/memcached.yml` is generated for each application on the environment.
+
+You need to add a memcached client to your app.
+
+## Installation
+
+For simplicity, we recommend that you create the `cookbooks/` directory at the
+root of your application. If you prefer to keep the infrastructure code separate
+from application code, you can create a new repository.
+
+Our main recipes have the `memcached_custom` cookbook but it is not included by default.
+To use the `memcached_custom` cookbook, you should copy the cookbook
+`custom-memcached`. You should not copy the actual `memcached_custom` recipe as
+that is managed by Engine Yard.
+
+1. Edit `cookbooks/ey-custom/recipes/after-main.rb` and add
+
+```
+    include_recipe 'custom-memcached'
+```
+
+2. Edit `cookbooks/ey-custom/metadata.rb` and add
+
+```
+    depends 'custom-memcached'
+```
+
+3. Copy `examples/memcached/cookbooks/custom-memcached` to `cookbooks/`
+
+```
+    cd ~ # Change this to your preferred directory. Anywhere but inside the
+         # application
+
+    git clone https://github.com/engineyard/ey-cookbooks-stable-v5
+    cd ey-cookbooks-stable-v5
+    cp examples/memcached/cookbooks/custom-memcached /path/to/app/cookbooks/
+```
+
+If you do not have `cookbooks/ey-custom` on your app repository, you can copy
+`examples/memcached/cookbooks/ey-custom` to `/path/to/app/cookbooks` as well.
+
```

Comment:
At some point the user should switch `perform_install` to `true`.  It is not necessarily a customization, so not belonging to that section of the Readme.